stable/sealed-secrets/templates/deployment.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "sealed-secrets.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "sealed-secrets.fullname" . }}
+    chart: {{ template "sealed-secrets.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  template:
+    metadata:
+      labels:
+        app: {{ template "sealed-secrets.fullname" . }}
+    spec:
+      {{ if .Values.rbacEnabled }}
+      serviceAccountName: {{ template "sealed-secrets.serviceAccountName" . }}
+      {{ end }}
+      containers:
+        - name: {{ template "sealed-secrets.fullname" . }}
+          command:
+            - controller
+          args: ["--key-name", "{{ .Values.secretName }}"]
+          image: quay.io/bitnami/{{ template "sealed-secrets.fullname" . }}:{{ .Values.image.tag }}
```

Comment:
Thank you for doing it @olib963 ! Unless I'm missing something obvious image should be hardcoded to `quay.io/bitnami/sealed-secrets-controller`, is not it?